ts/components/screens/BaseHeader.tsx

```diff
@@ -101,7 +101,7 @@ const setAccessibilityTimeout = 0 as Millisecond;
 const noReferenceTimeout = 150 as Millisecond;
 /** A component representing the properties common to all the screens (and the most of modal/overlay displayed) */
 class BaseHeaderComponent extends React.PureComponent<Props, State> {
-  private firstElementRef = React.createRef<NBText>();
+  private firstElementRef = React.createRef<any>(); // Could be NBText or View
```

Comment:
Can we use `<NBText | View>` instead of any and explicitly cast it later as
`ref={this.firstElementRef as React.RefObject<View>}`
?